crates/swc_css_minifier/tests/fixture/compress-calc/convert-units/output.min.css

```diff
@@ -1 +1 @@
-.class1{width:calc(1cm + 1px)}.class2{width:calc(1px + 1cm)}.class3{width:20q}.class4{width:calc(100.9q + 10px)}.class5{width:calc(10px + 100.9q)}.class6{width:calc(10cm + 1px)}.class7{width:calc(1cm + 1px)}.class8{width:calc(10px + 1q)}.class9{width:10.025cm}.class10{width:1.025cm}.class11{width:calc(10in + 1q)}.class12{width:calc(10pt + 1q)}.class13{width:calc(10pc + 1q)}.class14{width:calc(1q + 10px)}.class15{width:401q}.class16{width:41q}.class17{width:1017q}.class18{width:calc(1q + 10pt)}.class19{width:calc(1q + 10pc)}.class20{width:3unknown}.class21{width:calc(1unknown + 2px)}.class22{width:calc(1px + 2unknown)}
+.class1,.class2{width:calc(1cm + 1px)}.class3{width:20q}.class4,.class5{width:calc(10px + 100.9q)}.class6{width:calc(10cm + 1px)}.class7{width:calc(1cm + 1px)}.class8{width:calc(10px + 1q)}.class9{width:calc(10cm + 1q)}.class10{width:calc(1cm + 1q)}.class11{width:calc(10in + 1q)}.class12{width:calc(10pt + 1q)}.class13{width:calc(10pc + 1q)}.class14{width:calc(10px + 1q)}.class15{width:calc(10cm + 1q)}.class16{width:calc(1cm + 1q)}.class17{width:calc(10in + 1q)}.class18{width:calc(10pt + 1q)}.class19{width:calc(10pc + 1q)}.class20{width:3unknown}.class21{width:calc(2px + 1unknown)}.class22{width:calc(1px + 2unknown)}
```

Comment:
Some changes are unexpected, for example - `.class9{width:calc(10cm + 1q)}`, can you look at this?

Reply:
Fixed